src/esri-loader.ts

```diff
@@ -72,6 +79,11 @@ export const utils = {
 export function getScript() {
   return document.querySelector('script[data-esri-loader]') as HTMLScriptElement;
 }
+// TODO: export this function?
```

Comment:
I suppose leave it un-exported for now?

Reply:
I think for now, I want the `loadCss()` API to settle first